README.md

```diff
@@ -20,7 +20,7 @@ C++ Idiomatic Clients for [Google Cloud Platform][cloud-platform] services.
 [client-library-docs]: http://GoogleCloudPlatform.github.io/google-cloud-cpp/
 
 This library supports the following Google Cloud Platform services with clients
-at an [Alpha](#versioning) quality level:
+at an [Beta](#versioning) quality level:
```

Comment:
s/an/a/

Or, to be consistent with the other page, s/an/the/

Reply:
Fixed. Thanks.